### PR TITLE
[done] minor beautify changes

### PR DESCRIPTION
**Pull request by xlxs4**
### Some really really minor changes in order to achieve better consistency and layout

[genume-post-PR](https://imgur.com/gallery/WLWpNBt)

- [x] all variable's names comply to the same rules (e.g now all lowercase) when outputted
 - [**about tab**] scripts/about/**name.sh**: GENUME -> **genume**, Team -> **team** [a6acef4]
 - [**memory tab**] scripts/memory/**cache.sh**: num Of Caches -> **num of caches**
 - scripts/memory/**ram.sh**: memory total -> **total memory**, etc.

 - [x] general beautify changes
  - [**about tab**] scripts/about/**name.sh**: authors' list is now placed after current date and is displayed correctly (thanks to @MaanooAk & @damavrom [#69])
 - [**memory tab**] scripts/memory/**cache.sh**: when a cache didn't exist, various texts were outputted (does not exist, doesnt exist); now it outputs **<doesn't exist>** (the <> were added for them to be similar to the <opengl info not found> message in scripts/gpu/opengl.sh)
 - [**software tab**] scripts/software/**apps.sh**: removed the prefix app since adding it is just redundant
 - removed the missing apps group since it was both redundant and unnessecarily taking up space
 - now apps are grouped up for better readability

- [x] add more apps in scripts/software/**apps.sh**
**added so far**:
 - ghci, lua [0e45257] & [1cd570e]
 - vi / vim (previously the vi version check could output "vi: illegal option"), code, atom
 - okular, latex, freecad
 - i3
 - parted, stacer, transmission
 - vlc, ~~audacious~~, gimp, ~~kodi~~
 - chromium, firefox
 - thunderbird
 - zsh, csh, ksh, fish [f5e4c59]
 - shfmt, kiki, mnemosyne, thefuck, ghex, meld, ctags, ack
 - byobu, tmux, terminator
 - subversion
 - kde-applications, krename, kdiff
 - rvm, rbenv
 - wireshark
 - ssh, sshfs, filezilla, synergy, unison, sparkleshare
 - veracrypt, pgp, gpg
 - sublime, netbeans, eclipse, kate, codelite, gedit, geany, kwrite, scite
 - vagrant, docker
 - ranger, zathura, dia, gnuplot
 - htop
 - k3b
 - pidgin, hexchat

- [x] format script's code so that it abides with Google's styleguide to improve readability [4de4c36]
 - instead of 2 spaces use 4 for indentation [d9a19ec]

- [x] format all comments [394cb72]

### CODE REVIEW

File: scripts/software/apps.sh [hunk 1]
```diff
@@ -1,40 +1,243 @@
...
+check "ack" "--version"
+
+'''
+Terminal Management.
+'''
+
+# Terminal Multiplexer.
+check "byobu" "--version"
+# Another Terminal Multiplexer.
+check "tmux" "--version"
```
Comment (Sima214) [inline scripts/software/apps.sh]: [man tmux](http://man7.org/linux/man-pages/man1/tmux.1.html)

> -V            Report the tmux version.

File: scripts/software/apps.sh [hunk 2]
```diff
@@ -1,40 +1,243 @@
...
+
+'''
+Remote Connectivity.
+'''
+
+check "ssh" "-V"
+# Uses FUSE to mount folders remotely using SSH.
+check "sshfs" "--version"
+# FTP Client.
+check "filezilla" "--version"
```
Comment (Sima214) [inline scripts/software/apps.sh]: >Reading locale option from /home/sima/.config/filezilla/filezilla.xml
FileZilla 3.40.0, compiled on 2019-01-25

We may need to use grep to get the correct line.

File: scripts/software/apps.sh [hunk 3]
```diff
@@ -1,40 +1,243 @@
...
+
+# Terminal Multiplexer.
+check "byobu" "--version"
+# Another Terminal Multiplexer.
+check "tmux" "--version"
+# Multiple Terminals Management.
+check "terminator" "--version"
+
+'''
+# TODO:
```
Comment (Sima214) [inline scripts/software/apps.sh]: Part of the GNU toolchain.
Move to programming section.

File: scripts/software/apps.sh [hunk 4]
```diff
@@ -1,40 +1,243 @@
...
+check "freecad" "--version"
+
+'''
+Window Managers.
+'''
+
+check "i3" "--version"
+
+'''
+# TODO:
```
Comment (Sima214) [inline scripts/software/apps.sh]: parted: Add misc/unsorted section.
htop: System Monitors
stracer: System Monitors
transmission-cli: Personally I group these with my browsers. Could also go to misc/unsorted section.

File: scripts/software/apps.sh [hunk 5]
```diff
@@ -1,40 +1,243 @@
...
-		echo VALUE BAS "$key" \"$($name $version 2>&1 | sed '/./q; d')\" GROUP "apps"
-	else
-		echo VALUE BAS "$key" \""<not found>"\"	GROUP "missing_apps"
-	fi
+    if command -v "$name" >/dev/null; then
+        echo VALUE BAS "$key" \"$($name $version 2>&1 | sed '/./q; d')\" GROUP "apps"
+    fi
 }
 
+'''
```
[the remark this answers is not in the document]
Comment (xlxs4) [reply scripts/software/apps.sh]: Had no issue on my config somehow, will replace with shebang ones.

File: scripts/software/apps.sh (hunk 1 above)
Comment (xlxs4) [reply scripts/software/apps.sh]: What I get for being lazy and not checking them up :)
Thanks for pointing these out.

File: scripts/software/apps.sh (hunk 3 above)
Comment (xlxs4) [reply scripts/software/apps.sh]: The programming section consists of prog languages/compilers only. 
I think there should be a different distinct one for terminal emulators/management. 
An alternative suggestion could be grouping them up with the different shells, or with the window managers(?)

File: scripts/software/apps.sh (hunk 2 above)
Comment (Sima214) [reply scripts/software/apps.sh]: https://stackoverflow.com/a/23702386/7303580

Comment (xlxs4) [reply scripts/software/apps.sh]: Some pipes should do the trick:
**head -2 file | tail -1**                                       [FileZilla 3.40.0, compiled on 2019-01-25]
can also use awk, sed, cut and so on, for example:
**head -2 file | tail -1 | awk -F, '{ print $1 }'**      [FileZilla 3.40.0]

I'm leaving this up to you to decide on whichever suits best.

**Comment by xlxs4** [comment]
- **Styleguide** link: [0de6807]
- **Comments** Issue: [72094fc]
- Resolve TODOs related to **unsorted apps**: [a7a891c]

File: scripts/software/apps.sh (hunk 2 above)
Comment (Sima214) [reply scripts/software/apps.sh]: I was thinking of using regex which matches only the version string in 'check'. I mean to apply this rule globally.

File: scripts/software/apps.sh (hunk 4 above)
Comment (xlxs4) [reply scripts/software/apps.sh]: :heavy_check_mark:

File: scripts/software/apps.sh (hunk 3 above)
Comment (xlxs4) [reply scripts/software/apps.sh]: :heavy_check_mark:

File: scripts/software/apps.sh (hunk 1 above)
Comment (xlxs4) [reply scripts/software/apps.sh]: :heavy_check_mark:

File: scripts/software/apps.sh (hunk 5 above)
[the remark this answers is not in the document]
Comment (xlxs4) [reply scripts/software/apps.sh]: :heavy_check_mark:

File: scripts/software/apps.sh (hunk 2 above)
Comment (xlxs4) [reply scripts/software/apps.sh]: This sounds like the best solution there is. It's the simplest and will ensure homomorphism.

Comment (Sima214) [reply scripts/software/apps.sh]: https://www.gnu.org/software/grep/manual/grep.html#index-_002dP

> PCRE support is here to stay

It'll be fine...

Comment (Sima214) [reply scripts/software/apps.sh]: `pcregrep -o "(\d+\.)+\d+"` if we want to be sure about compatibility.

Comment (xlxs4) [reply scripts/software/apps.sh]: should I resolve the convos?

Comment (Sima214) [reply scripts/software/apps.sh]: Is it resolved?

Comment (xlxs4) [reply scripts/software/apps.sh]: check commits

Comment (Sima214) [reply scripts/software/apps.sh]: I still get the first line on the output.

Comment (Sima214) [reply scripts/software/apps.sh]: Click 'Resolve conversation' and I will merge this.

**Pull request merged**